Directory.Build.props

```diff
@@ -47,14 +47,14 @@
     <!-- Assembly/NuSpec attributes -->
     <Company>Microsoft</Company>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-    <PackageLicenseUrl>https://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</PackageLicenseUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/SignalR/SignalR/2.3.0/LICENSE.txt</PackageLicenseUrl>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(OfficialBuild)' != 'true'">
     <!-- Assembly/NuSpec attributes -->
     <Company>.NET Foundation</Company>
     <Copyright>© Copyright (c) .NET Foundation. All rights reserved.</Copyright>
-    <PackageLicenseUrl>https://raw.github.com/SignalR/SignalR/master/LICENSE.txt</PackageLicenseUrl>
```

Comment:
This change links to the 2.3.0 version of LICENSE.txt. In aspnet/AspNetCore, the PackageLicenseUrl always points to the 2.0.0 version of its LICENSE.txt, so there's precedent.

https://github.com/aspnet/AspNetCore/blob/e7b00a55083a56971e0d340ad2a3d350fec05883/Directory.Build.props#L14